removal_from_ansible.rst

```diff
@@ -0,0 +1,47 @@
+*****************************************************
+Ansible Community Package Collections Removal Process
+*****************************************************
+
+.. contents:: Topics
+
+Overview
+========
+
+This document describes under which circumstances collections can be removed from the `Ansible community package <https://pypi.org/project/ansible/>`_ (`build data <https://github.com/ansible-community/ansible-build-data/>`_). Right now this document tries to set up some first processes. In cases of emergency the `Ansible Community Engineering Steering Committee <https://github.com/ansible/community-docs/blob/main/ansible_community_steering_committee.rst>`_ can vote on emergency exceptions, but the goal is to only use processes documented in here.
+
+Removing broken collections
+===========================
+
+If it turns out that a collection contained in Ansible X.0.0 depends on another collection included in X.0.0 and does not work with it, the collection can be removed from Ansible (X+1).0.0 under the following conditions:
+
+1. The collection seems to be unmaintained, and nobody successfully manages to fix the problems.
```

Comment:
```suggestion
1. The collection seems to be unmaintained and nobody successfully manages to fix the problems.
```

Reply:
or maybe `nobody is trying to fix the problems` ? i.e. nobody is in active progress

Reply:
If folks try to fix it for two years and it doesn't work, it's way too long IMO. It's great that they try, but if they don't manage to do it, we still have to remove it even if they try continuously.